requirements.txt

```diff
@@ -2,3 +2,4 @@ click==8.1.3
 httpx==0.23.0
 pendulum==2.1.2
 pillow==9.2.0
+# error in devicy.py: backports.zoneinfo
```

Comment:
```suggestion
# if zoneinfo is not available: backports.zoneinfo
```
While you're at it, maybe specify the exact version, probably `backports.zoneinfo==0.2.1`.